roles/deploy/tasks/deploy.yml

```diff
@@ -2,10 +2,13 @@
 
 # Build codebase
 
-- name: remove the old build path
+- name: remove the old build paths
```

Comment:
should we name it `previous release paths`? we use the term `build` already for one of them, so it might lead to confusion.